src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -139,9 +139,9 @@ static bool UnionWithKeys(ConcurrentDictionary<string, bool> left, ConcurrentDic
         }
     }
 
-    private IQueryable<ResourceViewModel>? FilteredResources => _resourceByName.Values.Where(Filter).OrderBy(e => e.ResourceType).ThenBy(e => e.Name).AsQueryable();
+    private IQueryable<ResourceViewModel>? FilteredResources => _resourceByName.Values.Where(Filter).OrderBy(e => e.ResourceType).ThenBy(e => e, ResourceViewModelNameComparer.Instance).AsQueryable();
```

Comment:
Do we need to do this anywhere else?

Reply:
Console logs app selector could have be wrong as well. Fixed.